web/src/app/lists/FlatList.js

```diff
@@ -104,6 +105,7 @@ export default class FlatList extends React.PureComponent {
           primary={item.title}
           secondary={item.subText}
           secondaryTypographyProps={{ style: { whiteSpace: 'pre-line' } }}
+          inset={item.inset && !item.icon}
```

Comment:
Going off of MUI docs for this one, we needed an option to allow insetting the `ListItemText`, however their docs state that you should not inset the text if an icon is present:

```
inset | bool | false | If true, the children will be indented. This should be used if there is no left avatar or left icon.
```

So we are in fact doing that: the parent is setting all children to `inset: true`, however if the subscription is disabled (`icon` is set to display `<Warning />`), then the `inset` flag would be ignored

Reply:
That makes sense -- I'm mostly just wondering what cases we need to inset for specifically (I don't think I have all the context here), and if we need it on some list items and not others, within the same list, etc

I'm thinking:
- If we need both inset items, and non-inset (& without icon) in same list, than this is good as is.
- If it's only used when using icons (to keep things aligned) we could probably just "fix it everywhere" by calculating it
- If we want the entire list inset or not (while respecting the icon rule) we could potentially put inset on the parent once instead of needing to pass it into each item

Reply:
Ah, I see. The reasoning is because the prop is set on the list item text itself (rather than a global prop on the list), and we want each subscription item inset, so I added this as a prop to match the rest of the item props as those are also being applied via the item object